config/answers.capsule-installer.yaml

```diff
@@ -3,3 +3,21 @@ certs:
   generate: false
   deploy: true
 capsule: true
+foreman_proxy:
+  custom_repo: true
+  puppetca: true
+  http: true
+  http_port: '8000'
+  ssl_port: '9090'
+  register_in_foreman: true
```

Comment:
I think we can remove anything that's already default by Foreman: https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/params.pp